docs/template.md

```diff
@@ -1,7 +1,25 @@
-# qgrid 
+# qgrid template
 ## markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid rows="data" >
+        <q-grid:head> 
+            <q-grid:td> 
+                <i title="{{$column.name}}">{{$column.label}}</i>
+            </q-grid:td>
+        </q-grid:head>
+        <q-grid:body>
+            <q-grid:tr>
+            </q-grid:tr>
+            <q-grid:td key="firstName">
+                <b>{{$value}}</b>
+            </q-grid:td>
+            <q-grid:td key="dateBirth">
```

Comment:
What will be look of editable template? Something like triggers in wpf?
My variants:
*<q-grid:td-in-editmode>
*<q-grid:td:edit>
*<q-grid:td on='edit'>
*<q-grid:td when='$mode === "edit"'>
*<q-grid:td on:edit='true'>
*<q-grid:td on:mode='edit'>   ?